.travis.yml

```diff
@@ -27,5 +27,5 @@ before_script:
   - tools/travis/before_build_byond.sh
 
 script:
-  - tools/travis/check_filedirs.sh ColonialMarinesALPHA.dme
```

Comment:
different PR for this please

Reply:
It's some merge garbage which crawled in here. Pretty much already merged stuff from different PRs.
Perhaps it may be because I didn't properly name the conflict resolving commit, but may have no impact on the actual code afaik.

Reply:
still here